### DISCUSSION TITLE
如何禁用 emoji，或者是否可以使用颜文字代替 emoji？

### DISCUSSION THREAD

**Discussion by sommio** [fcitx/fcitx5#1180] [General]
找到唯一一个设置「emoji」的地方位于「键盘」里，但似乎不起作用

![image](https://github.com/user-attachments/assets/e6ee713a-1c0d-43c8-a010-b1e6b23397a6)
![image](https://github.com/user-attachments/assets/2e08243c-a374-4b2c-a955-8f7fe683edd6)

**Comment by wengxt** [comment]
在拼音的设置里面找 `启用符号`
